src/main/java/io/vertx/openapi/validation/analyser/ContentAnalyser.java

```diff
@@ -50,6 +50,22 @@ public Object transform() {
     }
   }
 
+  private static class TextPlainAnalyser extends ContentAnalyser {
```

Comment:
This is a duplicate of OctetStreamAnalyser. If multiple types require a no op analyser, we should rename the analyser and re-use it.

Reply:
I had this idea as well. Totally agree with you.